src/resize/resize.go

```diff
@@ -0,0 +1,146 @@
+// Package resize implements algorithms for resizing images
+// and contains some helpers for working with files and creating simple images.
+package resize
+
+import (
+	"image"
+	"math"
+
+	"image/color"
+	"image/draw"
+)
+
+// Naive changes image size to specified width and height.
+// Function uses an algorithm similar to nearest-neighbour interpolation width similar results.
+// Width and height have to be more than zero.
+func Naive(src image.Image, width int, height int) image.Image {
+	if src == nil {
+		return nil
+	}
+	dst := createCanvas(src, width, height)
+	naiveResize(src, dst, width, height)
+	return dst
+}
+
+func naiveResize(src image.Image, dst draw.Image, width int, height int) {
```

Comment:
In function `bilinearResize` you use check
``` 
if src == nil || dst == nil {
		return
	}
```
Should you use it in this method ?